configextractor_/configextractor_.py

```diff
@@ -259,12 +267,29 @@ def execute(self, request):
                 if network_section:
                     parser_section.add_subsection(network_section)
 
+                for binary in config.get('binaries', []):
+                    # Append binary data to submission for analysis
+                    datatype = binary.get('datatype', 'other')
+                    data = binary.get('data')
+                    if datatype in ['other', 'payload'] and data:
+                        if isinstance(data, str):
+                            data = data.encode()
+                        sha256 = hashlib.sha256(data).hexdigest()
+                        a = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
You could use **with** to close the file at the end, and should probably use the self.working_directory. I am not certain you need the seek(0) after the write?